src/lib.rs

```diff
@@ -209,64 +186,33 @@ impl Dimensions {
 pub struct Grid {
     options: GridOptions,
     cells: Vec<Cell>,
-    widest_cell_length: Width,
-    width_sum: Width,
-    cell_count: usize,
+    widest_cell_length: usize,
```

Comment:
To be consistent with `Cell` where the corresponding property is named `width`: 
```suggestion
    widest_cell_width: usize,
```